### PR TITLE
Issue777 pending forward ref

### PR DESCRIPTION
**Pull request by ekr**

### CODE REVIEW

File: draft-ietf-rtcweb-jsep.xml [hunk 1]
```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
         answerer, who first calls setRemoteDescription(sdp [offer]) and
         then later setLocalDescription(sdp [answer]).</t>
 
+        <t>During the offer/answer exchange (i.e., in states other than
+        stable), the outstanding offer is "pending" at the offerer
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: "stable" has not yet been defined at this point in the document.

Also suggest the phrasing 'is considered to be "pending"'

File: draft-ietf-rtcweb-jsep.xml [hunk 2]
```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
...
+        and the answerer and may either be accepted or rejected. If this
+        is a re-offer, each side will also have "current" local and
+        remote descriptions, which reflect the state of the last
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggestion: state -> result

File: draft-ietf-rtcweb-jsep.xml [hunk 3]
```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
         answerer, who first calls setRemoteDescription(sdp [offer]) and
         then later setLocalDescription(sdp [answer]).</t>
 
+        <t>During the offer/answer exchange (i.e., in states other than
+        stable), the outstanding offer is "pending" at the offerer
+        and the answerer and may either be accepted or rejected. If this
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Suggestion: "... answerer, as it may either ..."

File: draft-ietf-rtcweb-jsep.xml (hunk 1 above)
Comment (ekr) [inline draft-ietf-rtcweb-jsep.xml]: Yeah, what do you think I should do for the stable thing? We could shove this text after the state machine...

Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: I would just omit the parenthetical. It's pretty clear what's meant here.

**Comment by ekr** [comment]
@jubert: PTAL

File: draft-ietf-rtcweb-jsep.xml [hunk 4]
```diff
@@ -3096,14 +3109,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
...
 
-        <t>Note that the application MAY modify the SDP to reduce the
-        capabilities in the offer it sends to the far side
-        (post-setLocalDescription) or the offer that it installs from
-        the far side (pre-setRemoteDescription), as long as it remains
-        a valid SDP offer and specifies a subset of what was in the
-        original offer. This is safe because the answer is not
-        permitted to expand capabilities, and therefore will just
-        respond to what is present in the offer.</t>
+        <t>After calling setLocalDescription, the application MAY modify the SDP to
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: Remove this

File: draft-ietf-rtcweb-jsep.xml [hunk 5]
```diff
@@ -319,6 +319,19 @@ ipr="trust200902">
         answerer, who first calls setRemoteDescription(sdp [offer]) and
         then later setLocalDescription(sdp [answer]).</t>
 
+        <t>During the offer/answer exchange,
+        the outstanding offer is considered to be pending at the offerer
```
Comment (juberti) [inline draft-ietf-rtcweb-jsep.xml]: pending -> "pending"

**Pull request merged**